### PR TITLE
ADMINUI-2291 Search jobs by job UUID was removed

### PR DESCRIPTION
**Pull request by Arc311**

### CODE REVIEW

File: www/js/views/jobs.jsx [hunk 1]
```diff
@@ -269,6 +275,18 @@ var JobsView = Backbone.Marionette.Layout.extend({
                 delete filter.params[key];
             } else if ((key === 'until' || key === 'since') && typeof value === 'object') {
                 filter.params[key] = moment(value).utc().toDate().getTime();
+            } else if (key === 'name' && /^[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}$/.test(filter.params[key])) {
```
Comment (YZaicev) [inline www/js/views/jobs.jsx]: regex to CONST

**Comment by YZaicev** [comment]
🆗 🔢

File: www/js/views/jobs.jsx [hunk 2]
```diff
@@ -269,6 +276,18 @@ var JobsView = Backbone.Marionette.Layout.extend({
...
+                var self = this;
+                var job = new Job({
+                    uuid: filter.params.name
+                });
+                this.listenToOnce(job, 'error', self.jobsList.onError);
+                return job.fetch().done(function () {
+                    self.jobsList.collection.reset();
+                    self.jobsList.collection.set([job]);
+                }).fail(function () {
+                    self.jobsList.collection.reset();
```
Comment (dan-luk) [inline www/js/views/jobs.jsx]: can this be in .finally?

**Comment by dan-luk** [comment]
🔢

**Comment by askfongjojo** [comment]
committed as 088a619

**Pull request closed without merging**